style.css

```diff
@@ -1,18 +1,763 @@
+}
+
+.fivestars__safety{
+display: flex;
+flex-direction: row;
+margin-top: 80px;
+}
+
+.fivestars__shield{
+width: 62px;
+height: 74px;
+}
+
+.fivestars__textstars{
+position: relative;
+margin-left: 10px;
+margin-top: 0;
+margin-bottom: 0;
+font-style: normal;
+font-weight: 600;
+font-size: 50px;
+line-height: 30px;
+}
+
+.fivestars__span{
+font-style: normal;
+font-weight: 600;
+font-size: 25px;
+}
+
+.fivestars__active{
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+margin-top: 30px;
+}
+
+.fivestars__menu{
+display: flex;
+font-style: normal;
+font-weight: bold;
+font-size: 13px;
+line-height: 16px;
+text-align: center;
+text-transform: uppercase;
+color: #FFFFFF;
+text-decoration: none;
+}
+
+.fivestars__menu_active{
+border: 2px solid #FFFFFF;
+width: 160px;
+height: 40px;
+display: flex;
+align-items: center;
+justify-content: center;
+margin-right: 10px;
+}
+
+.fivestars__link{
+display: flex;
+align-items: center;
+font-style: normal;
+font-weight: bold;
+font-size: 13px;
+line-height: 16px;
+text-align: center;
+text-transform: uppercase;
+color: #FFFFFF;
+text-decoration: none;
+}
+
+.fivestars__circle{
+margin-right: 10px;
+}
+
+.schema{
+position: relative;
+width: 70%;
+height: 100%;
+display: flex;
+flex-direction: column;
+}
+
+.schema__title{
+position: relative;
+display: flex;
+width: 80%;
+top: 100px;
+left: 17%;
+}
+
+.schema__imgcar{
+display: flex;
+position: relative;
+width: 100%;
+}
+
+
+
+@media (max-width:1440px){
+	body{
+	max-width: 1400px;
+	}
+
+	.header{
+		max-width: 1400px;
+	}
+	.tesla{
+		max-width: 1400px;
+	}
+}
+
+@media (max-width:992px){
+	body{
+	max-width: 768px;
+	}
+
+	.header{
+		max-width: 768px;
+	}
+
+	.menu{
+		display: none;
+	}
+
+	.menu__store{
+		display: none;
+	}
+	.tesla{
+		max-width: 768px;
+	}
+	.specifications{
+		flex-direction: column;
+		align-items: center;
+	}
+
+	.specifications__arrow{
+		margin-top: 5%;
+	}
+
+	.salon{
+		flex-direction: column;
+	}
+
+	.salon__text{
+		align-items: flex-end;
+		right: 50px;
+	}
+
+	.description{
+		position: relative;
+		flex-direction: row;
+		flex-wrap: wrap;
+		width: 100%;
+		align-items: center;
+		height: 100%;
+	}
+
+	.parameters__info{
+		margin-bottom: 2%;
+	}
+
+	.safety{
+		flex-direction: column-reverse;
+	}
+
+	.safety__block{
+		position: relative;
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: space-around;
+		width: 100%;
+	}
+
+	.schema{
+		display: flex;
+		align-items: center;
+		width: 100%;
+	}
+
+	.schema__imgcar{
+		width: 90%;
+	}
+
+	.schema__title{
+		position: absolute;
+		width: 70%;
+		top: 0;
+	}
+
+	.specifications__item{
+		color: #E51937;
+	}
+
+	.specifications__itemlink{
+		color: #FFFFFF;
+		border: 2px solid #E51937;
+		background-color: #E51937;
+	}
+
+	.specifications__steering{
+		display: none;
+	}
+
+	.specifications__steeringred{
+		display: flex;
+		width: 25px;
+		height: 25px;
+	}
+}
+
+
+@media (max-width:767px){
+	body{
+	max-width: 375px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)